cmake/windows-setup.cmake

```diff
@@ -3,114 +3,113 @@
 # Not all plugins are supported on Windows yet. This file tweaks
 # the build flags so that we can compile fluent-bit on it.
 
-set(FLB_REGEX                 Yes)
-set(FLB_BACKTRACE              No)
-set(FLB_LUAJIT                Yes)
-set(FLB_EXAMPLES              Yes)
-set(FLB_PARSER                Yes)
-set(FLB_TLS                   Yes)
-set(FLB_AWS                   Yes)
-set(FLB_HTTP_SERVER           Yes)
-set(FLB_METRICS               Yes)
+if(FLB_WINDOWS_DEFAULTS)
```

Comment:
I don't think you need this twice, we already guard the include of this file and this change just means a load of whitespace change - plus people may want to include it themselves directly.

Reply:
This is the option for the default values, only Plugins shall be affected by this option.

Compiler options, Flex/Bison, and config file must be done anyway for windows

Reply:
Sorry, my point was this whole file is only included when `FLB_WINDOWS_DEFAULTS` is set - there is no point checking it again inside the file.

Reply:
is it clear now?

Reply:
It's clear to me, at first I got confused because of the similarity in the names but right before I sent a comment asking for further clarification I got it, thanks!